src/machines/java/com/enderio/machines/common/init/MachineBlocks.java

```diff
@@ -159,7 +159,7 @@ public class MachineBlocks {
     public static final Map<SolarPanelTier, BlockEntry<SolarPanelBlock>> SOLAR_PANELS = Util.make(() -> {
```

Comment:
Should we rename the solar tiers to match the new recipe/capacitor banks?

Reply:
What do you mean by this?